agent/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package agent runs tasks on build hosts. It polls the API server for a new
+// task, gets the configuration for that task, and then runs it.
```

Comment:
I think the second sentence isn't that relevant to the agent overall, unless we wanted to describe the different components of the agent in greater detail.

Does this deserve more explanation?